Makefile

```diff
@@ -73,12 +91,17 @@ install.tools: .install.gitvalidation .install.glide .install.glide-vc .install.
 	gometalinter --install --update
 
 clean:
-	rm -rf *~ $(OUTPUT_DIRNAME) $(TOOLS)
+	rm -rf *~ $(OUTPUT_DIRNAME) $(TOOLS) $(MAN)
 
 .PHONY: \
+	all \
 	tools \
-	$(TOOLS) \
+	man \
+	install \
+	uninstall \
 	check-license \
 	clean \
 	lint \
-	test
+	test \
+	$(MAN) \
+	$(TOOLS)
```

Comment:
Neither `$(MAN)` nor `$(TOOLS)` are phony.  They are all real files that we're building, and they all have targets pointing at them.